### ISSUE TITLE
Focal T2 hyperintense substantia nigra lesion

### ISSUE THREAD

**Issue by MLK6014** [obophenotype/human-phenotype-ontology#6454]
**Preferred term label:**
Focal T2 hyperintense substantia nigra lesion

**Synonyms**
Hyperintense lesion in the substantia nigra on MRI

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
HP:0007183: Focal T2 hyperintense basal ganglia lesion

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Leigh Syndrome #256000

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as Focal substantia nigra T2 hyperintensity
